src/dotnet/commands/dotnet-crossgen/Exceptions/CrossGenException.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Microsoft.DotNet.Tools.CrossGen.Exceptions
```

Comment:
It seems odd to me to put this single class in its own namespace.  Can it just go in `Microsoft.DotNet.Tools.CrossGen`?